hangman.py

```diff
@@ -83,10 +88,16 @@ def get_available_letters(letters_guessed):
     returns: string (of letters), comprised of letters that represents which letters have not
       yet been guessed.
     '''
-    # FILL IN YOUR CODE HERE AND DELETE "pass"
-    pass
-    
-    
+    alphabet = string.ascii_lowercase
+    new_alp = ''
+    for letter in alphabet:
+        if letter in letters_guessed:
+            new_alp += ''
+        else:
+            new_alp += f'{letter}'
+
+    return new_alp
+
 
 def hangman(secret_word):
```

Comment:
This function is too long and consist mostly from code. Please - add some comments and extract some methods here

Reply:
I tried, but code doesnt work as well as should. So i have no solution for now

Reply:
It means that you need to debug your code, it's not a reason to give up)

Reply:
Creted a new method letter_check() which make validation of entered letter